lib/email_alert_criteria.rb

```diff
@@ -12,7 +12,6 @@ def would_trigger_alert?
     return false unless english?
     return false unless change_note?
 
-    return false if blocked_publishing_app?
```

Comment:
What's the verdict? Should this change be fixed up or reinstated?

Reply:
Have fixed up now (partially reinstated to block collections-publisher only, drug_safety_alert blocked in document types).